tests/hmm/hmumu_utils.py

```diff
@@ -436,12 +447,14 @@ def analyze_data(
                 dnn_vars["lumi"] = scalars["luminosityBlock"][dnn_presel]
                 dnn_vars["event"] = scalars["event"][dnn_presel]
                 dnn_vars["dnn_pred"] = dnn_prediction
-                dnn_vars["dnnPisa_pred"] = dnnPisa_prediction
+                if not (len(dnnPisa_predictions)==0):
+                    for imodel in range(4):
```

Comment:
here you could also get this `4` from parameters.

Reply:
removed the hard-coded "4"